_third_party/github.com/StackExchange/httpunit/httpunit.go

```diff
@@ -393,6 +393,7 @@ type TestResult struct {
 	GotText     bool
 	GotRegex    bool
 	InvalidCert bool
+	TimeTotal   time.Duration
```

Comment:
You shouldn't edit files in _third_party directly. Instead this feature will need to be a seperate pr to github.com/StackExchange/httpunit